source/Libraries/GogLibrary/GogLibrarySettingsView.xaml

```diff
@@ -17,8 +17,16 @@
     </UserControl.Resources>
 
     <StackPanel Margin="20">
+        <StackPanel Orientation="Horizontal">
+            <TextBlock VerticalAlignment="Center" Text="Language:" Margin="0,0,10,0"/>

```

Comment:
1) Should IMO say "metadata language" to be more clear what it affects.
2) Please put it at the bottom of all settings, it's not more important than connecting account.
3) The string should be translatable.